src/main/resources/cucumber-junit-re-runner.vm

```diff
@@ -0,0 +1,171 @@
+import cucumber.runtime.Runtime;
+import cucumber.runtime.io.MultiLoader;
+import cucumber.runtime.io.ResourceLoaderClassFinder;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.junit.Test;
+import com.github.temyers.options.ExtendedRuntimeOptions;
+import java.io.*;
+import java.util.*;
+
+
+public class $className{
+
+        private String outputPath="$outputPath";
+        private String glue =$glue;
+
+        @Test
+        public void reRun()  {
+
+                defaultRun();
+
+                int count= $retryCount;
+
+                System.out.println("Retry Count specified is :- "+count);
+
+                switch (count){
```

Comment:
I can add it in loop. But its meaningless to keep reruning without looking for root cause of failure. Test case won't pass magically so i kept it initially 5 rerun.